src/Tabs/Tab/Tab.component.tsx

```diff
@@ -0,0 +1,154 @@
+// VENDOR
+import * as React from 'react';
+import classNames from 'classnames';
+import styled, { css, ThemeContext } from '@xstyled/styled-components';
+import { th, space as spaceStyles } from '@xstyled/system';
+// ANCHOR
+import { Position, Variant, variantStyles } from '../utils';
+
+interface TabProps extends React.HTMLAttributes<HTMLDivElement> {
+    className?: string;
+
+    position?: Position;
+    active?: boolean;
+    disabled?: boolean;
+    activeBorder?: string;
+    activeBorderColor?: string;
+
+    spacing?: string;
+    offset?: string;
+    border?: string;
+    margin?: string;
+    padding?: string;
+    background?: string;
+    borderRadius?: string;
+
+    variant?: Variant;
+}
+
+interface StyledTabProps extends TabProps {
+    tabTheme?: any;
+}
+
+const StyledTab = styled('div')<StyledTabProps>`
+    display: flex;
+    cursor: pointer;
+    user-select: none;
+    position: relative;
+
+    transition: 250ms ease background, 250ms ease color, 125ms ease border-color;
+
+    &.disabled {
+        color: text.disabled;
+        cursor: not-allowed;
+    }
+
+    ${({ tabTheme }) => tabTheme.styles}
```

Comment:
Idk if it's just my editor but all of these arrow functions throw lint errors if they don't end with semi-colons.